test/server/cards/01_SOR/units/BodhiRook.spec.ts

```diff
@@ -27,12 +27,12 @@ describe('Bodhi Rook', function () {
                 });
                 expect(context.player1).not.toHaveEnabledPromptButton('Done');
 
-                // Check that cards are not revealed in chat
-                expect(context.getChatLogs(1)[0]).not.toContain(context.sabineWren.title);
-                expect(context.getChatLogs(1)[0]).not.toContain(context.battlefieldMarine.title);
-                expect(context.getChatLogs(1)[0]).not.toContain(context.infernoFour.title);
-                expect(context.getChatLogs(1)[0]).not.toContain(context.waylay.title);
-                expect(context.getChatLogs(1)[0]).not.toContain(context.protector.title);
+                // Check that cards are revealed in chat
+                expect(context.getChatLogs(1)[0]).toContain(context.sabineWren.title);
+                expect(context.getChatLogs(1)[0]).toContain(context.battlefieldMarine.title);
+                expect(context.getChatLogs(1)[0]).toContain(context.infernoFour.title);
+                expect(context.getChatLogs(1)[0]).toContain(context.waylay.title);
+                expect(context.getChatLogs(1)[0]).toContain(context.protector.title);
```

Comment:
Might not be a bad idea to update these tests to be explicit about the log message they expect, now that we actually _do_ expect a log message. Similar to the one for Qi'ra below.